tools/opensnoop.py

```diff
@@ -169,12 +173,15 @@
 """
 
 bpf_text_kprobe_header_open = """
-int syscall__trace_entry_open(struct pt_regs *ctx, const char __user *filename, int flags)
+int syscall__trace_entry_open(struct pt_regs *ctx, const char __user *filename,
+                              int flags, unsigned short mode)
```

Comment:
The mode for 'open' should be an int type, right?
See https://man7.org/linux/man-pages/man3/mode_t.3type.html